components/automate-ui/src/app/modules/infra-proxy/data-bags-list/data-bags-list.component.spec.ts

```diff
@@ -46,7 +47,8 @@ describe('DataBagsListComponent', () => {
         ReactiveFormsModule,
         RouterTestingModule,
         StoreModule.forRoot(ngrxReducers, { runtimeChecks })
-      ]
+      ],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
Several test files were missing the CUSOM_ELEMENTS_SCHEMA and as a result were throwing linting errors.